security/target_path.rst

```diff
@@ -17,36 +17,42 @@ the user is redirected back to a page which the browser cannot render.
 To get around this behavior, you would simply need to extend the ``ExceptionListener``
 class and override the default method named ``setTargetPath()``.
 
-First, override the ``security.exception_listener.class`` parameter in your
-configuration file. This can be done from your main configuration file (in
-``app/config``) or from a configuration file being imported from a bundle:
+First, add a ``CompilerPass`` which you will use to override the ``ExceptionListener`` class::
```

Comment:
I used override because that's the terminology from here: https://symfony.com/doc/current/bundles/override.html
But happy to change it :)